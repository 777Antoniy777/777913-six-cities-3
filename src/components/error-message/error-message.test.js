import React from 'react';
import renderer from "react-test-renderer";
import ErrorMessage from "./error-message";

// set mocha data
const requestMessage = `message`;
const wrapperStyle = {};
const messageStyle = {};

it(`render ErrorMessage`, () => {
  const tree = renderer.create(
      <ErrorMessage
        requestMessage={requestMessage}
        wrapperStyle={wrapperStyle}
        messageStyle={messageStyle}
      />)
      .toJSON();

  expect(tree).toMatchSnapshot();
});
