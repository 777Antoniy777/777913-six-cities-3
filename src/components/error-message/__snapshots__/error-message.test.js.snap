// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render ErrorMessage 1`] = `
<div
  style={Object {}}
>
  <p
    style={Object {}}
  >
    message
  </p>
</div>
`;
